subsys/mgmt/mcumgr/Kconfig

```diff
@@ -123,6 +123,15 @@ config FS_MGMT_PATH_SIZE
 	  and download commands.
 endif
 
+config MCUMGR_CMD_SHELL_MGMT
+	bool "Enable mcumgr handlers for shell management"
+	depends on SHELL
+	select SHELL_BACKEND_DUMMY
+	help
+	  Enables mcumgr handlers for shell management. Must use the dummy backend
```

Comment:
`Must use the dummy backend ...` -> `The handler utilizes the dummy shell back-end to capture (...)`

or something like that, reword it little bit.

Reply:
Makes sense. Will do. Thanks!

Reply:
I believe this is all taken care of now. I just updated the language according to your recommendations, and added a bit more clarifying detail.